sdk/keyvault/azure-security-keyvault-jca/README.md

```diff
@@ -0,0 +1,127 @@
+        System.getProperty("azure.tenant.id"), 
+        System.getProperty("azure.client.id"),
+        System.getProperty("azure.client.secret"));
+    ks.load(parameter);
+
+    KeyManagerFactory kmf = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+    kmf.init(ks, "".toCharArray());
+
+    SSLContext context = SSLContext.getInstance("TLS");
+    context.init(kmf.getKeyManagers(), null, null);
+
+    SSLServerSocketFactory factory = (SSLServerSocketFactory) context.getServerSocketFactory();
+    SSLServerSocket serverSocket = (SSLServerSocket) factory.createServerSocket(8765);
+```
+
+Note if you want to use Azure managed identity, you should set the value
+of `azure.keyvault.uri`, and the rest of the parameters would be `null`.
+
+## Client side SSL
+
+If you are looking to integrate the JCA provider for client side socket 
+connections, see the Apache HTTP client example below.
+
+```java
+    KeyVaultJcaProvider provider = new KeyVaultJcaProvider();
+    Security.addProvider(provider);
+
+    KeyStore ks = KeyStore.getInstance("AzureKeyVault");
+    KeyVaultLoadStoreParameter parameter = new KeyVaultLoadStoreParameter(
+            System.getProperty("azure.keyvault.uri"), 
+            System.getProperty("azure.tenant.id"), 
+            System.getProperty("azure.client.id"),
+            System.getProperty("azure.client.secret"));
+    ks.load(parameter);
+
+    SSLContext sslContext = SSLContexts
+            .custom()
+            .loadTrustMaterial(ks, new TrustSelfSignedStrategy())
+            .build();
+
+    SSLConnectionSocketFactory sslSocketFactory = SSLConnectionSocketFactoryBuilder
+            .create()
+            .setSslContext(sslContext)
+            .setHostnameVerifier((hostname, session) -> {
+                return true;
+            })
+            .build();
+
+    PoolingHttpClientConnectionManager cm = PoolingHttpClientConnectionManagerBuilder
+            .create()
+            .setSSLSocketFactory(sslSocketFactory)
+            .build();
+
+    String result = null;
+
+    try ( CloseableHttpClient client = HttpClients.custom().setConnectionManager(cm).build()) {
+        HttpGet httpGet = new HttpGet("https://localhost:8766");
+        HttpClientResponseHandler<String> responseHandler = (ClassicHttpResponse response) -> {
+            int status = response.getCode();
+            String result1 = "Not success";
+            if (status == 204) {
+                result1 = "Success";
+            }
+            return result1;
+        };
+        result = client.execute(httpGet, responseHandler);
+    } catch (IOException ioe) {
+        ioe.printStackTrace();
+    }
+```
+
+Note if you want to use Azure managed identity, you should set the value
+of `azure.keyvault.uri`, and the rest of the parameters would be `null`.
+
+## Spring Boot
+
+For Spring Boot applications see our [Spring Boot starter]<!--(../../spring/azure-spring-boot-starter-keyvault-certificates/README.md)-->.
```

Comment:
Does this link need to be set to go somewhere?

Reply:
This will be fixed in a subsequent PR as it is referring to a location that is currently not published

Reply:
Fixed